src/types/types.ts

```diff
@@ -905,11 +905,15 @@ export interface EdgeCurrencyEngineOptions {
   userSettings: JsonObject | undefined
 }
 
+export interface StartEngineOpts {
+  lightMode?: boolean
+}
```

Comment:
Please move this flag to `makeCurrencyEngine`, since this applies to the entire engine, whether it's running or not.